client/blocks/comment-detail/style.scss

```diff
@@ -0,0 +1,261 @@
+%comment-detail__ellipsis {
+	overflow: hidden;
+	text-overflow: ellipsis;
+	white-space: nowrap;
+}
+
+%comment-detail__gray-link {
+	color: $gray;
+	cursor: pointer;
+	fill: $gray;
+	&:focus,
+	&:hover {
+		color: $blue-wordpress;
+		fill: $blue-wordpress;
+	}
+}
+
+.comment-detail.card {
+	font-size: 14px;
+	margin: 0 auto;
+	padding: 0;
+	transition: margin .15s linear;
+
+	&.is-expanded {
+		margin: 16px auto;
+	}
+	&:not( .is-expanded ):hover {
+		box-shadow: 0 0 0 1px $gray, 0 2px 4px lighten( $gray, 20% );
+		z-index: 1;
+	}
+
+	.gridicon {
+		margin-right: 4px;
+		vertical-align: middle;
+	}
+}
+
+.comment-detail__author-avatar {
+	flex-shrink: 0;
+	img {
+		border-radius: 50%;
+	}
+}
+.comment-detail__author-info {
+	@extend %comment-detail__ellipsis;
```

Comment:
I suppose we do suggest using sass `extend` in our css guidelines. While `extend` does make the unminified file smaller, usually you’re usually better off just repeating yourself since the gzipped file will likely be smaller. (gzip is great at compressing repeated strings).

Note that we currently can't use extend across media queries, so if we think we'll be adding any, we should avoid this. https://github.com/sass/sass/issues/1050

cc @mtias if you had other thoughts